ckanext/sitesearch/logic/chained_action.py

```diff
@@ -1,6 +1,35 @@
 from ckan.plugins import toolkit
 
-from ckanext.sitesearch.lib import index
+from ckanext.sitesearch.logic.action import package_groups
+from ckanext.sitesearch.lib import index, rebuild
+
+
+@toolkit.chained_action
+def package_create(up_func, context, data_dict):
+
+    data_dict = up_func(context, data_dict)
+
+    owner_org = data_dict.get("owner_org", None)
+    if owner_org:
+        rebuild.rebuild_orgs(entity_id=owner_org)
+
+    return data_dict
+
+
+@toolkit.chained_action
+def package_delete(up_func, context, data_dict):
+    package_id = toolkit.get_or_bust(data_dict, "id")
+    groups = package_groups(context.copy(), {"id": package_id})
```

Comment:
The whole `package_groups` function seems overkill (specially as we are dictizing all groups/orgs and you just need the group/org id and is_organization, so maybe we just use ` groups = pkg.get_groups()` and drop the whole function